src/main/java/com/minecolonies/coremod/entity/ai/citizen/builder/ConstructionTapeHelper.java

```diff
@@ -103,8 +51,9 @@ public static void placeConstructionTape(@NotNull AbstractBuilding building, @No
 
     /**
      * Place construction tape.
+     *
      * @param workOrder the workorder.
-     * @param world the world.
+     * @param world     the world.
      */
 
     public static void placeConstructionTape(@NotNull WorkOrderBuild workOrder, @NotNull World world)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Refactor this method to reduce its Cognitive Complexity from 28 to the 20 allowed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3776)